### PR TITLE
Updated the UI to the new design system

### LINKED ISSUE #343: Update UI as per the new design system
@goutham-subramanyam _a Please work on the UI changes and let's follow a similar convention like we followed for neetoUI.

cc: @AbhayVAshokan

### PR DESCRIPTION
**Pull request by goutham-subramanyam**
Fixes #343

**Description**
- Changed: updated UI to the new design system.

**Checklist**

- [x] I have performed a self-review of my code.
- [x] ~~I have made corresponding changes to the documentation.~~
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).
- [x] I have followed the suggested description format and styling.

**Reviewers**

### CODE REVIEW

**Comment by goutham-subramanyam** [comment]
@karthiknmenon _a Please check and merge.

cc: @AbhayVAshokan

File: lib/components/Common/Avatar.jsx [hunk 1]
```diff
@@ -2,12 +2,13 @@ import React, { useState } from "react";
 
 import classNames from "classnames";
 import PropTypes from "prop-types";
+import { isNil } from "ramda";
 
 const SIZE = {
```
Comment (karthiknmenon) [inline lib/components/Common/Avatar.jsx:7]: Can we move this along with all other constants to a constant file?

**Review by karthiknmenon — CHANGES_REQUESTED** [review]
@goutham-subramanyam _a Please make the minor changes as discussed over call.

**Review by AbhayVAshokan — COMMENTED** [review]
The `yarn build` command is failing. @goutham-subramanyam _a can you please make this change at:
https://github.com/bigbinary/neeto-editor/blob/77d5098e9141e51bad09c158650ab9b138653e57/rollup.config.js#L46
```diff
-       commonjs()
+       commonjs({ include: /\**node_modules\**/ })
```

Also, once done, can you please commit the output of the `yarn build` command? This would generate the updated `editor-content.min.css` file which would be served to the changelog widget via neetoEditor CDN.

PS: This change is directly taken from [neetoUI#rollupConfig](https://github.com/bigbinary/neeto-ui/blob/e58a6fe859646d959ce160156b3a6d7726e554cb/rollup.config.js#L38).

**Comment by goutham-subramanyam** [comment]
@karthiknmenon _a Made the suggested changes. Please check.

cc: @AbhayVAshokan

**Review by karthiknmenon — APPROVED** [review]
LGTM.

@AbhayVAshokan _a Please confirm and merge.

**Comment by AbhayVAshokan** [comment]
Screenshot 2022-09-26 at 6 41 14 PM

Screenshot 2022-09-26 at 6 41 28 PM

@goutham-subramanyam I am not a big fan of large buttons. I was curious to know if large buttons are indeed the right choice for these small modals.

cc: @karthiknmenon

**Comment by goutham-subramanyam** [comment]
@AbhayVAshokan I had missed that. I have updated the buttons.

**Review by AbhayVAshokan — APPROVED** [review]
LGTM ❤️

**Pull request merged**